docs/feature-checklist.md

```diff
@@ -0,0 +1,422 @@
+Instead of rewriting our own feature checklist from scratch, we make a derivative of the
+[Verus checklist](https://github.com/verus-lang/verus/blob/main/source/docs/guide/src/features.md)
+which is available for use under [MIT](https://github.com/verus-lang/verus/blob/main/LICENSE) 
+license.
+
+**Last Updated: 2025-04-07**
+
+<div class="table-wrapper"><table>
+  <thead><tr><th colspan="2"><strong>Items</strong></th></tr></thead>
+  <tbody>
```

Comment:
I would much prefer for this table to be a set of markdown tables. We don't need the HTML code and it will get in the way when we update the table.
What cannot be done is the merged cells as sub-headers. we can do separate tables for each as one option